samples/vmc/sddc/deploy_ovf_template_cloud.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+"""
+* *******************************************************
+* Copyright VMware, Inc. 2018. All Rights Reserved.
+* SPDX-License-Identifier: MIT
+* *******************************************************
+*
+* DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+* EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+* WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+* NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+"""
+
+__author__ = 'VMware, Inc.'
+__vcenter_version__ = 'VMware Cloud on AWS'
+
+from com.vmware.content.library_client import Item
+from com.vmware.vcenter.ovf_client import LibraryItem
+from com.vmware.vcenter_client import ResourcePool
+from vmware.vapi.vsphere.client import create_vsphere_client
+
+from samples.vsphere.common import sample_cli, sample_util
+from samples.vsphere.common.id_generator import generate_random_uuid
+from samples.vsphere.contentlibrary.lib.cls_api_helper import ClsApiHelper
+
+
+class DeployOvfTemplateCloud:
+    """
+    Demonstrates the workflow to deploy an OVF library item to
+    a resource pool in VMware Cloud on AWS.
+    Note: the sample needs an existing library item with an OVF template
+    and an existing resource pool with resources for deploying the VM.
+    """
+
+    def __init__(self):
+        parser = sample_cli.build_arg_parser()
+
+        parser.add_argument('--resource_pool_name',
```

Comment:
nit: Can we name the arguments consistently? libitemname and resourcepoolname or resource_pool_name and item_name? 

In the CL VMTX samples (deploy_vm_template), the argument name is resourcepoolname.

Reply:
Changed to resourcepoolname